pkg/queue/manager_test.go

```diff
@@ -153,6 +154,93 @@ func TestAddWorkload(t *testing.T) {
 	}
 }
 
+func TestStatus(t *testing.T) {
+	ctx := context.Background()
+	scheme := runtime.NewScheme()
+	if err := kueue.AddToScheme(scheme); err != nil {
+		t.Fatalf("Failed adding kueue scheme: %s", err)
+	}
+	now := time.Now().Truncate(time.Second)
+
+	queues := []kueue.Queue{
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "foo"},
+			Spec: kueue.QueueSpec{
+				Capacity: "fooCap",
+			},
+		},
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "bar"},
+			Spec: kueue.QueueSpec{
+				Capacity: "barCap",
+			},
+		},
+	}
+	workloads := []kueue.QueuedWorkload{
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:              "a",
+				CreationTimestamp: metav1.NewTime(now.Add(time.Hour)),
+			},
+			Spec: kueue.QueuedWorkloadSpec{QueueName: "foo"},
+		},
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:              "b",
+				CreationTimestamp: metav1.NewTime(now),
+			},
+			Spec: kueue.QueuedWorkloadSpec{QueueName: "bar"},
+		},
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:              "c",
+				CreationTimestamp: metav1.NewTime(now),
+			},
+			Spec: kueue.QueuedWorkloadSpec{QueueName: "foo"},
+		},
+	}
+	manager := NewManager(fake.NewClientBuilder().WithScheme(scheme).Build())
+	for _, q := range queues {
+		if err := manager.AddQueue(ctx, &q); err != nil {
+			t.Errorf("Failed adding queue: %s", err)
+		}
+	}
+	for _, wl := range workloads {
+		wl := wl
+		manager.AddOrUpdateWorkload(&wl)
+	}
+	wantStatus := make(map[string]int32)
+	wantStatus["foo"] = 2
+	wantStatus["bar"] = 1
+
+	status := make(map[string]int32)
+	statusFoo, err := manager.Status(&queues[0])
+	if err != nil {
+		t.Errorf("Failed retrieving queue status: %s", err)
+	}
+	status[queues[0].Name] = int32(statusFoo)
+	statusBar, err := manager.Status(&queues[1])
+	if err != nil {
+		t.Errorf("Failed retrieving queue status: %s", err)
+	}
+	status[queues[1].Name] = int32(statusBar)
+	if diffReport := cmp.Diff(wantStatus, status); diffReport != "" {
```

Comment:
```suggestion
	if diff := cmp.Diff(wantStatus, status); diff != "" {
```

Just to be consistent with the rest of the codebase